packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -32,27 +31,27 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
-    removeRefIdPrefix = (ref_id) => {
-        const splitRefId = ref_id.split('xccdf_org.ssgproject.content_profile_')[1];
+    removeRefIdPrefix = (refId) => {
+        const splitRefId = refId.toLowerCase().split('xccdf_org.ssgproject.content_profile_')[1];
```

Comment:
Unfortunately I noticed that remediations for the C2S do not work without making the identifier lowercase, because calling remediations with `ssg:rhel7|C2S|xccdf_org.ssgproject.content_rule_ensure_gpgcheck_never_disabled` does not work. It has to be `c2s`